### PR TITLE
alinosratipour-class7-javascript-week1

### PR DESCRIPTION
**Pull request by alinosratipour**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: mandatory/1-syntax-errors.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-    return a + b + c;
+// you can not return param on its own if you expecting some results.
+function addNumbers(a, b, c) {
+  let sum = a + b + c;
```
Comment (kmh256) [inline mandatory/1-syntax-errors.js:5]: Check the indentation

File: mandatory/1-syntax-errors.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-    total = a ++ b;
 
-    return "The total is total"
-}
+// ++ has no meaning also you can not just retune string we are expecting result.
+ function getTotal(a, b) {
+    let total = a + b;
+    let result = total;
+    let msg = "The total is " + result ;
+     return msg;
```
Comment (kmh256) [inline mandatory/1-syntax-errors.js:21]: Check the indentation

**Review by kmh256 — COMMENTED** [review]
Good work Ali! I have left a couple of comments :)

**Comment by Laura-TM** [comment]
I think we were meant to follow a template for naming our pull request (e.g. London Class 7 - Your Name - Javascript Core 1 - Week 1) along with the details in the description field.

**Pull request closed without merging**